packages/documentation/gatsby-config.js

```diff
@@ -71,14 +71,6 @@ module.exports = {
     },
     `gatsby-plugin-sass`,
     `gatsby-plugin-sitemap`,
-    {
-      resolve: `gatsby-source-filesystem`,
-      options: {
-        path: `../formation-react/src/components`,
-        name: 'formation-react',
-        ignore: ['**/*.stories.*'],
-      },
-    },
```

Comment:
This is being removed because the files are no longer used in `createFormationReactPages`